.travis.yml

```diff
@@ -2,7 +2,9 @@ language: java
 jdk:
   - oraclejdk8
 addons:
-  sauce_connect: true
+  sauce_connect:
+    username: alloyui
```

Comment:
Until now we only have this problem on Android platforms, but there is a similar issue related to iOS in another project.

https://support.saucelabs.com/customer/portal/questions/14577625-hide-the-safari-self-signed-certificate-warning